python_modules/dagster/dagster/_core/definitions/run_status_sensor_definition.py

```diff
@@ -757,6 +759,14 @@ def _wrapped_fn(
                     else:
                         job_match = True
 
+                # check if the job was created by `materialize()` and was therefore ephemeral
+                if (
+                    not job_match:
```

Comment:
this looks like a typo to me

Can you run "make ruff" and "make pyright" in the repo before resubmitting